packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,415 @@
+//TODO all file upload process needed to be checked and refactored with TS. skipping checks for passing CFD build
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React from 'react';
+import { Formik, FormikErrors, FormikHelpers, FormikValues } from 'formik';
+import { DocumentUploadResponse } from '@deriv/api-types';
+import {
+    Loading,
+    Button,
+    FormSubmitErrorMessage,
+    Text,
+    ThemedScrollbars,
+    FormSubmitButton,
+    Modal,
+} from '@deriv/components';
+import { isMobile, validAddress, validPostCode, validLetterSymbol, validLength, getLocation, WS } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import FormFooter from '../../../Components/form-footer';
+import FormBody from '../../../Components/form-body';
+import FormBodySection from '../../../Components/form-body-section';
+import FormSubHeader from '../../../Components/form-sub-header';
+import LoadErrorMessage from '../../../Components/load-error-message';
+import LeaveConfirm from '../../../Components/leave-confirm';
+import FileUploaderContainer from '../../../Components/file-uploader-container';
+import CommonMistakeExamples from '../../../Components/poa/common-mistakes/common-mistake-examples';
+import PersonalDetailsForm from '../../../Components/forms/personal-details-form.jsx';
+import { validate } from '../../../Helpers/utils';
+import { isServerError } from '../../../Types/common.type';
+
+const FilesDescription = () => (
+    <div className='files-description'>
+        <Text size={isMobile() ? 'xxs' : 'xs'} as='div' className='files-description__title' weight='bold'>
+            {localize(
+                'We accept only these types of documents as proof of your address. The document must be recent (issued within last 6 months) and include your name and address:'
+            )}
+        </Text>
+        <ul>
+            <li>
+                <Text size={isMobile() ? 'xxs' : 'xs'}>
+                    {localize('Utility bill: electricity, water, gas, or landline phone bill.')}
+                </Text>
+            </li>
+            <li>
+                <Text size={isMobile() ? 'xxs' : 'xs'}>
+                    {localize(
+                        'Financial, legal, or government document: recent bank statement, affidavit, or government-issued letter.'
+                    )}
+                </Text>
+            </li>
+            <li>
+                <Text size={isMobile() ? 'xxs' : 'xs'}>
+                    {localize('Home rental agreement: valid and current agreement.')}
+                </Text>
+            </li>
+        </ul>
+    </div>
+);
+
+export type TFileRef = React.RefObject<null | { upload: () => Promise<DocumentUploadResponse> }> | undefined;
+
+let file_uploader_ref: TFileRef;
+
+type TProofOfAddressForm = {
+    is_resubmit: boolean;
+    is_qualified_for_cfd_modal?: boolean;
```

Comment:
ProofOfAddressForm is of type Partial<TProofOfAddressForm>. Do we need to set is_qualified_for_cfd_modal as optional again?

Reply:
No @amina-deriv, thnaks